lib/commands/condition.js

```diff
@@ -109,6 +168,30 @@ function requireConditionInducerCompatibleDevice() {
   }
 }
 
+/**
+ * @param {string} udid
+ * @returns {Promise<DVTServiceWithConnection>}
+ */
+async function startRemoteXPC(udid) {
+  const {Services} = await import('appium-ios-remotexpc');
+  return Services.startDVTService(udid);
+}
+
+/**
+ * @param {XCUITestDriver} driver
+ * @returns {Promise<void>}
+ */
+async function closeRemoteXPC(driver) {
```

Comment:
make `driver` to `this`, similarly to how requireConditionInducerCompatibleDevice is designed